src/model/index.js

```diff
@@ -10,7 +10,7 @@ export default {
     async (_, { cartId, customerId, customerToken }, { dispatch }) => {
       await dispatch.cart.getCart(cartId)
       await dispatch.cart.setCartId(cartId)
-      // await dispatch.user.setCustomerId(customerId)
```

Comment:
@petro97 there's lots of leftover commented code here. I'd prefer we don't keep it if it's not needed